ui/raidboss/data/07-dt/raid/r4n.ts

```diff
@@ -1,14 +1,274 @@
+      netRegex: { id: ['9B4F', '9B55'], source: 'Wicked Thunder', capture: false },
+      run: (data) => data.expectedBlasts = 4,
+    },
+    {
+      id: 'R4N Fivefold Blast Initializer',
+      type: 'StartsUsing',
+      netRegex: { id: ['9B56', '9B57'], source: 'Wicked Thunder', capture: false },
+      run: (data) => data.expectedBlasts = 5,
+    },
+    {
+      id: 'R4N XFold Blast Collector',
+      type: 'GainsEffect',
+      netRegex: { effectId: 'B9A', count: Object.values(effectB9AMap), capture: true },
+      condition: (data, matches) => {
+        const count = matches.count;
+
+        if (!isEffectB9AValue(count))
+          return false;
+        data.storedBlasts.push(count);
+
+        return data.storedBlasts.length >= data.expectedBlasts;
+      },
+      durationSeconds: (data) => {
+        if (data.expectedBlasts === 3)
+          return 14.4;
+        if (data.expectedBlasts === 4)
+          return 18.9;
+        return 23.2;
+      },
+      infoText: (data, _matches, output) => {
+        const expectedBlasts = data.expectedBlasts;
+        data.expectedBlasts = 3;
+        const storedBlasts = data.storedBlasts;
+        data.storedBlasts = [];
+        switch (expectedBlasts) {
+          case 3:
+            return output.three!({
+              first: b9aValueToNorthSouth(storedBlasts[0]),
+              second: b9aValueToNorthSouth(storedBlasts[1]),
+              third: b9aValueToNorthSouth(storedBlasts[2]),
+            });
+          case 4:
+            return output.four!({
+              first: b9aValueToNorthSouth(storedBlasts[0]),
+              second: b9aValueToNorthSouth(storedBlasts[1]),
+              third: b9aValueToNorthSouth(storedBlasts[2]),
+              fourth: b9aValueToNorthSouth(storedBlasts[3]),
+            });
+          case 5:
+            return output.five!({
+              first: b9aValueToNorthSouth(storedBlasts[0]),
+              second: b9aValueToNorthSouth(storedBlasts[1]),
+              third: b9aValueToNorthSouth(storedBlasts[2]),
+              fourth: b9aValueToNorthSouth(storedBlasts[3]),
+              fifth: b9aValueToNorthSouth(storedBlasts[5]),
+            });
+          default:
+            return output.unknown!();
+        }
+      },
+      outputStrings: {
+        south: Outputs.south,
+        north: Outputs.north,
+        unknown: Outputs.unknown,
+        three: {
+          en: '${first} => ${second} => ${third}',
+        },
+        four: {
+          en: '${first} => ${second} => ${third} => ${fourth}',
+        },
+        five: {
+          en: '${first} => ${second} => ${third} => ${fourth} => ${fifth}',
+        },
+      },
+    },
+    {
+      id: 'R4N Bewitching Flight Right Safe',
+      type: 'StartsUsing',
+      netRegex: { id: '8DE4', source: 'Wicked Thunder', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'East offset safe',
```

Comment:
Not sure what 'offset' means in this context - I got a little confused when I saw this pop up on screen (same for the other 3 below).  There are lines safe or gid squares safe, so not sure what a direction being safe necessarily means.  (Also, the trigger title might be confusing too.)

Reply:
The "lines" vs "squares" doesn't work when taking into consideration the second clone, which can cause it to be "on the horizontal line but off the vertical line". Also IIRC lines vs squares only works on the middle-most lines, but I'd have to double check footage to be sure.

I'm open to better descriptions here as long as they make sense with regards to the east clone. Alternatively, these triggers could be disabled for now to get the others out if we want to let the wording here cook a bit longer.

Reply:
Yeah, let's keep thinking on it.  Not sure 'offset' will make sense to some people, but I don't have a better suggestion right now.